tests/mallinfo.cpp

```diff
@@ -29,7 +29,7 @@ extern "C" {
 
 unsigned int getTotalMemory()
 {
-	return EM_ASM_INT(return TOTAL_MEMORY);
```

Comment:
Technically this can be breaking change if any user code does this kind of thing, right?

Reply:
It is, yeah - but this is an internal implementation detail we've never documented as stable (or documented at all, I think ;)